tests/unit/utils/envVars.go

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import (
+	coreV1 "k8s.io/api/core/v1"
+)
+
+func FindEnvVarByName(haystack []coreV1.EnvVar, needle string) *coreV1.EnvVar {
```

Comment:
While this made me laugh, it might be confusing to other users in the future. Maybe change these to something like `envVars` and `name` or `envVarName`.

Reply:
Heh, that was a leftover from when I was trying to write it generically. 

FWIW, I've used this nomenclature in the past (in Java), and never had it confuse anyone. It's both whimsical and very obvious. :)

Nonetheless, since it's not generic in any way, it makes sense to change it here.